File: BOJ/Recommended1/Bronze/16561/16561.py
# n을 일단 3으로 나누기 range(1,1001)
n = int(input())
cnt = 0
for x in range(3,n-1,3) :
    for y in range(3,n-1,3) :
        if n-x-y <=0  : continue
        cnt+=1


print(cnt)

File: BOJ/Recommended1/Bronze/16561/16561.swift
let n = Int(readLine()!)!
var cnt = 0
for x in stride(from: 3, through: n-1, by: 3) {
    for y in stride(from: 3, through: n-1, by: 3) {
        if n-x-y <= 0  {continue}
        cnt+=1
    }
}
print(cnt)